ports/esp32/boards/ARDUINO_NANO_ESP32/pins.csv

```diff
@@ -12,6 +12,14 @@ D10,GPIO21
 D11,GPIO38
 D12,GPIO47
 D13,GPIO48
+D17,GPIO1
+D18,GPIO2
+D19,GPIO3
+D20,GPIO4
```

Comment:
✅  Single digit GPIO pins are listed without a proceeding zero (e.g. `GPIO4` as opposed to `GPIO04`). This matches both:
- The [ESP 32](https://www.espressif.com/sites/default/files/documentation/esp32_datasheet_en.pdf) datasheet (pg 12):
![image](https://github.com/sebromero/micropython/assets/75624145/f03f6b41-18fb-4fd9-b83d-9f1408e28bba)

- Our Arduino [Nano ESP 32 documentation](https://docs.arduino.cc/tutorials/nano-esp32/pin-setup/#nano-esp32-pin-map):
![image](https://github.com/sebromero/micropython/assets/75624145/6a4cfe64-2f46-4cbc-a30b-cc3b606eeebf)